### PR TITLE
Support 0.100.0+ config

### PR DESCRIPTION
**Pull request by aprasolova**

### CODE REVIEW

File: defaults/main/main.yml [hunk 1]
```diff
@@ -124,9 +125,24 @@ neogo__default_config:
...
+
+  - enabled: 'True'
+    path: ApplicationConfiguration
+    options:
+      P2P:
+        Addresses: '{{ neogo__p2p_addresses }}'
+        DialTimeout: 3s
+        ProtoTickInterval: 2s
+        PingInterval: 30s
+        PingTimeout: 90s
```
Comment (532910) [inline defaults/main/main.yml:142]: let's put all strings in quotes

File: defaults/main/main.yml [hunk 2]
```diff
@@ -64,19 +68,25 @@ neogo__rpc_start_when_synchronized: False
...
 neogo__tls_port: '{{ neogo__network_type_settings[neogo__network_type].tls | d() }}'
+neogo__tls_addresses:
+  - '{{ neogo__tls_address }}:{{ neogo__tls_port }}'
 neogo__tls_cert_file: '{{ neogo__conf_dir }}/server.crt'
 neogo__tls_key_file: '{{ neogo__conf_dir }}/server.key'
 
 neogo__prometheus_enabled: False
 neogo__prometheus_address: 'localhost'
 neogo__prometheus_port: 2112
+neogo__prometheus_addresses:
```
Comment (532910) [inline defaults/main/main.yml:79]: I'd prefer to see an arrays with a single element defined as `[]`

File: defaults/main/network_opts.yml [hunk 3]
```diff
@@ -61,7 +61,7 @@ neogo__default_network_config:
         - 'seed4.neo.org:10333'
         - 'seed5.neo.org:10333'
       Hardforks:
-        Aspidochelone: 1730000
+        Aspidochelone: 2800000
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply defaults/main/network_opts.yml:64]: No. This can break mainnet, in fact. Where this value is coming from?

File: defaults/main/main.yml [hunk 4]
```diff
@@ -47,13 +47,17 @@ neogo__bind_address: '0.0.0.0'
 # Ports and options for each service
 neogo__p2p_address: '{{ neogo__bind_address }}'
 neogo__p2p_port: '{{ neogo__network_type_settings[neogo__network_type].p2p }}'
```
Comment (roman-khimov) [inline defaults/main/main.yml:49]: Why keeping old address/port pairs?

File: defaults/main/main.yml [hunk 5]
```diff
@@ -124,9 +125,24 @@ neogo__default_config:
   - enabled: '{{ neogo__wallet_struct|length > 0 }}'
     path: ApplicationConfiguration
     options:
-      UnlockWallet:
-        Path: '{{ neogo__wallet_path }}'
-        Password: '{{ neogo__wallet_password }}'
+      Consensus:
+        Enables: True
```
Comment (roman-khimov) [inline defaults/main/main.yml:129]: There is no `Enables` there.

File: defaults/main/main.yml [hunk 6]
```diff
@@ -124,9 +125,24 @@ neogo__default_config:
   - enabled: '{{ neogo__wallet_struct|length > 0 }}'
```
Comment (roman-khimov) [inline defaults/main/main.yml:125]: Maybe we can have some `consensus` config? The idea is exactly to not have and abstract `wallet`. There are services, and each of them can use a wallet of its own.

**Pull request merged**